### DISCUSSION TITLE
Add an option to disable row_limit

### DISCUSSION THREAD

**Discussion by GregNas** [apache/superset#18483] [Ideas]
Request for an option to disable row_limit.
Today there is only an option to select the number of rows to limit but there is no option to disable it.
There are databases like Sybase ASE and Sybase IQ that not supports the "limit" dialect. 

It can be row_limit = note in the drop-down list or any other option to disable force addition of limit to the SQL syntax.

**Comment by Kaibu** [comment]
Would love to use superset aswell, but it's just not possible with Sybase ASE due to this issue..
Seems like the interest to fix this is low tho.

**Comment by E-Babkin** [comment]
Please provide a way to configure the syntax of select limits in Superset.
LIMIT/OFFSET clause is far from the standard, and many popular DBMs use other syntax variants.
https://troels.arvin.dk/db/rdbms/#select-limit

**Comment by HosniAkremi** [comment]
This will be essential in order to query OpenLink Virtuoso RDBMS.
Cheers,
